CMakeLists.txt

```diff
@@ -32,33 +32,39 @@ project(hpipm)
 
 set(CMAKE_VERBOSE_MAKEFILE ON)
 
-# Target architecture
-set(TARGET GENERIC)
+# Options
 
 # BLAS and LAPACK version (for LA=BLAS in BLASFEO)
-set(REF_BLAS 0)
-#set(REF_BLAS OPENBLAS)
-#set(REF_BLAS NETLIB)
-#set(REF_BLAS MKL)
-#set(REF_BLAS BLIS)
-#set(REF_BLAS ATLAS)
+set(REF_BLAS 0 CACHE STRING "Set CPU architecture target")
+set(REF_BLAS OPENBLAS CACHE STRING "Set CPU architecture target")
+set(REF_BLAS NETLIB   CACHE STRING "Set CPU architecture target")
+set(REF_BLAS MKL      CACHE STRING "Set CPU architecture target")
+set(REF_BLAS BLIS     CACHE STRING "Set CPU architecture target")
+set(REF_BLAS ATLAS    CACHE STRING "Set CPU architecture target")
 
-# Options
 # set(HPIPM_TESTING ON CACHE BOOL "Tests enabled")
 set(HPIPM_TESTING OFF CACHE BOOL "Tests disabled")
 
-# BLASFEO
-if(NOT TARGET blasfeo)
-	# BLASFEO installation directory
-	set(BLASFEO_PATH /opt/blasfeo)
+# Target architecture
+set(TARGET GENERIC CACHE STRING "Set CPU architecture target")
```

Comment:
I guess we can put `HASWELL` as default by now

Reply:
Also in `Makefile.rule` is `AVX`

Reply:
Fixed in latest commits.